module/VuFind/src/VuFind/Search/Base/Options.php

```diff
@@ -1524,6 +1541,9 @@ protected function configureAutocomplete(?array $searchSettings = null)
         if (null !== ($autosubmit = $autocompleteSettings['auto_submit'] ?? null)) {
             $this->autocompleteAutoSubmit = $autosubmit;
         }
+        if (null !== ($suggestionlimit = $autocompleteSettings['suggestion_limit'] ?? null)) {
+            $this->autocompleteSuggestionLimit = (int) $suggestionlimit;
```

Comment:
Very minor style point: our code doesn't usually put a space after a typecast.

```suggestion
            $this->autocompleteSuggestionLimit = (int)$suggestionlimit;
```

Reply:
And now that the merge conflict is resolved, the other GitHub Actions have run, and they're complaining about this too -- that's why it didn't get caught before my first review. You can run `composer fix` on your system and commit any resulting changes to take care of this kind of style issue automatically.